src/generic/splitter.cpp

```diff
@@ -71,17 +71,37 @@ wxEND_EVENT_TABLE()
 
 bool wxSplitterWindow::AlwaysUsesLiveUpdate() const
 {
-    return !wxClientDC::CanBeUsedForDrawing(this);
+    return false;
 }
 
 static bool IsLive(wxSplitterWindow* wnd)
 {
     // with wxSP_LIVE_UPDATE style the splitter windows are always resized
     // following the mouse movement while it drags the sash, without it we only
     // draw the sash at the new position but only resize the windows when the
-    // dragging is finished -- but drawing the sash is done using wxClientDC,
-    // so check if we can use it and always use live resizing if we can't
-    return wnd->AlwaysUsesLiveUpdate() || wnd->HasFlag(wxSP_LIVE_UPDATE);
+    // dragging is finished
+    return wnd->HasFlag(wxSP_LIVE_UPDATE);
+}
+
+static wxBitmap wxPaneCreateStippleBitmap()
```

Comment:
We should extract this function somewhere and probably cache the bitmap because even if it's just a 2x2 one, I am still not sure we want to recreate it on every repaint.

The trouble is that we need to clean it up in some module and I don't see anything obviously appropriate. Perhaps we could make this a stock brush? Then it would be cleaned up by `wxStockGDI::DeleteAll()`.

But at least let's factor out this function into some private header, it's not quite trivial and it might need to be changed later (for dark mode or high DPI or I don't know what else) and so I'd rather not have 2 copies of it.

Reply:
Updating the documentation and this refactoring are still not done yet. if you don't mind me asking you to do that when testing under wxOSX, or fixing any regressions introduced (which I haven't noticed any!). TIA!

Reply:
Would it be better to do the refactoring in a seperate PR instead?